pkg/kubelet/eviction/helpers.go

```diff
@@ -779,7 +779,7 @@ func thresholdsMet(thresholds []evictionapi.Threshold, observations signalObserv
 		threshold := thresholds[i]
 		observed, found := observations[threshold.Signal]
 		if !found {
-			glog.Warningf("eviction manager: no observation found for eviction signal %v", threshold.Signal)
+			glog.V(4).Infof("eviction manager: no observation found for eviction signal %v", threshold.Signal)
```

Comment:
Yes, it would be nice if we only let the user know about this if it is a new condition i.e. we just started and couldn't find the observation or we have seen the observation before but it has been unavailable for some reason.

I might make a follow-on PR to do this.